### PR TITLE
Incorporated new name attribute field from node registry

### PR DESCRIPTION
**Pull request by alexandercyu**
- Updated load-node-content.js and node.js to use new name attribute from node registry
- Changed "name" variable in  load-node-content.js, ide.html, datacatalog.html to "nodeName" to be less generic
- Changed "name" id to "display_name" to be less generic

### CODE REVIEW

File: templates/site/datacatalog.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
                 })
             }
 
-            function setNodeLink(name, data, node_id_suffix) {
+            function setNodeLink(nodeName, data, node_id_suffix) {
```
Comment (mishaschwartz) [inline templates/site/datacatalog.html:31]: This isn't really the node's name anymore though. It's a unique id that may or may not have anything to do with the name.

Calling this variable `nodeName` is going to be confusing. I'd change it to `nodeID` (and elsewhere)

File: templates/site/js/load-node-content.js [hunk 2]
```diff
@@ -15,12 +15,12 @@ function binNodes(registry, binSize) {
...
 
-function buildNodeDescription(name, data, node_index, row_id_suffix) {
+function buildNodeDescription(nodeName, data, node_index, row_id_suffix) {
     let node_id_suffix = `${row_id_suffix}-${node_index}`;
     const node_template = document.getElementById("link-description").cloneNode(true)
-    let href = `node.html?node=${name}`;
+    let href = `node.html?node=${nodeName}`;
     let img_src;
-    let img_alt = `${name} icon`;
+    let img_alt = `${nodeName} icon`;
```
Comment (mishaschwartz) [inline templates/site/js/load-node-content.js:23]: image alts are user facing so this should be the node name not the id:

```javascript
let img_alt = `${data.name} icon`;
```

File: templates/site/node.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
                 <div class="row">
                     <div class="col-9">
                         <!--H2 title -->
-                        <div id="name" class="mb-0"></div>
+                        <div id="display_name" class="mb-0"></div>
```
Comment (mishaschwartz) [inline templates/site/node.html:11]: I don't think this div is ever filled with anything.... is changing the id necessary? Should we get rid of this div or is there supposed to be something here?

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
Looks good. Just some confusion with the variable naming

File: templates/site/node.html (hunk 3 above)
Comment (alexandercyu) [reply templates/site/node.html:11]: I don't remember what it was originally for.  Best I can guess is it's supposed to fill in the node's name for the title (Red Oak@University of Toronto).  

If it's not used I'll take it out.

File: templates/site/datacatalog.html (hunk 1 above)
Comment (alexandercyu) [reply templates/site/datacatalog.html]: Changed to nodeID

File: templates/site/js/load-node-content.js (hunk 2 above)
Comment (alexandercyu) [reply templates/site/js/load-node-content.js]: Changed to the node's name

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**